integ/failure-server/Dockerfile.toxy

```
@@ -0,0 +1,4 @@
+FROM node:12-stretch
+COPY ./toxy/index.js ./toxy/yarn.lock ./toxy/package.json ./
+RUN yarn
+ENTRYPOINT yarn start
```

For some reason the no-newline icon bothers me.

```suggestion
ENTRYPOINT yarn start

```